cli/jobs/automl-standalone-jobs/cli-automl-distributed-classification-task-creditcard/cli-distributed-classification.yaml

```diff
@@ -0,0 +1,36 @@
+$schema: https://azuremlsdk2.blob.core.windows.net/preview/0.0.1/autoMLJob.schema.json
+type: automl
+
+experiment_name: cli_classification_task_distributed_lightgbm
+# name: dpv2-cli-regression-train-job-01
+description: A classification job using credit card dataset
+
+compute: azureml:automl-distributed-cluster
+
+task: classification
+log_verbosity: debug
+primary_metric: accuracy
+
+target_column_name: "Class"
+
+training_data:
+  path: https://automlsamplenotebookdata.blob.core.windows.net/automl-sample-notebook-data/creditcard-training-mltable
+  type: mltable
+
+validation_data:
+  path: https://automlsamplenotebookdata.blob.core.windows.net/automl-sample-notebook-data/creditcard-validation-mltable
+  type: mltable
+
+limits:
+  timeout_minutes: 120
+  max_trials: 5
+  max_concurrent_trials: 4
```

Comment:
If I am correct, in the distributed TCN the $\text{max nodes} = \text{max concurrent trials} \times \text{factor}$. Max nodes is 6, max concurrent should be either 3 or 2 to have a multiple of 2 and 3, respectively.

Reply:
for classification/regression, `max_nodes` means the number of nodes to use for each concurrent trial, I believe. So here, you're asking for 6 nodes for each trial * 4 concurrent trials = 24 nodes in total. At least, this is what the API docs say

Reply:
For classifciation/regression max_concurrent_trials is ignored. 
Currently we always use concurrency of 1

Reply:
so let us remove max_concurrent_trials setting from our notebooks for classification/regression

Reply:
> For classifciation/regression max_concurrent_trials is ignored. Currently we always use concurrency of 1

Vinod, if concurrency is 1. This means every iteration of LightGBM uses the `max_nodes` to train. If a user wants 2 iterations of LightGBM, they cannot be run concurrently. The second iteration starts after the first is completed. Is my understanding correct?

Reply:
@vlbejan Minimum nodes that we need to run distributed LightGBM classification/regression task should be 4. My understanding is same as yours. @vinodshanbhag can you please explain a bit more here?

Reply:
@vlbejan you are right. That is the current limitation. 
We could allow higher concurrency in the future.. but keeping concurrency to 1 is Ok for now.

Reply:
@Nivi09 In this case we should remove the `max_concurrency` since it is not really an option for the user and reduce the size of `max_nodes` to 4 with appropriate explanation.

Reply:
Ah good to know - I didn't know multiple concurrent trials weren't supported. I'll update my public doc PR with a note about this

Reply:
Done. Thank you.